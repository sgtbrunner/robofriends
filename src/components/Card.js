import React from 'react';

const Card = ({ name, email }) => {
 	return (
   <div className="dib br3 pa1 ma2 grow bw2 shadow-5 bg-light-green">
     <img alt="robot" src={`https://robohash.org/${name}`} />
     <h2>{name}</h2>
     <p>{email}</p>
   </div>
 	);
};

export default Card;
