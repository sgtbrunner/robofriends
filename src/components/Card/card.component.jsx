import React from 'react';

const Card = ({ name, email, id }) => (
  <div className="dib br3 pa1 ma2 grow bw2 shadow-5 bg-light-green">
    <img alt="robot" src={`https://robohash.org/${name + email + id}`} />
    <h2>{name}</h2>
    <p>{email}</p>
  </div>
);

export default Card;
