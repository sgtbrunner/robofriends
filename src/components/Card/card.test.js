import React from 'react';
import { shallow } from 'enzyme';
import Card from './card.component';

it('expect to render Card component', () => {
  expect(shallow(<Card />).debug()).toMatchSnapshot();
});
