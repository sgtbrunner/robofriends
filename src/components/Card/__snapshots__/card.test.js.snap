// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
"<div className=\\"dib br3 pa1 ma2 grow bw2 shadow-5 bg-light-green\\">
  <img alt=\\"robot\\" src=\\"https://robohash.org/NaN\\" />
  <h2 />
  <p />
</div>"
`;
